Replicator/IncomingRev.hh

```diff
@@ -48,6 +48,8 @@ namespace litecore { namespace repl {
 
         int progressNotificationLevel() const override;
 
+        uint64_t bodySizeOfRemoteSequence() const { return _bodySizeOfRemoteSequence; }
```

Comment:
This should not be exposed as a public method.

Reply:
The parent puller will have to call it. Make Puller a friend?

Reply:
If it's called by the puller, wouldn't it be appropriate in the section marked "Called by the Puller" then?

Reply:
yes, I will do it.

Reply:
It doesn't need to be called by the Puller at all. The IncomingRev should pass the `_bodySize` to the Puller.